app/build.gradle

```diff
@@ -8,7 +8,7 @@ android {
         applicationId "ai.elimu.appstore"
         minSdkVersion 21
         targetSdkVersion 26
-        versionCode 2000009
+        versionCode 2000010
```

Comment:
Do not upgrade the versionCode without making a corresponding change to the versionName.